.github/workflows/update-win-version.yml

```diff
@@ -0,0 +1,74 @@
+name: Update Windows distribution version
+
+on:
+  workflow_call:
+    inputs:
+      vs19_vcpkg_version:
+        required: true
+        type: string
+      app_version:
+        required: true
+        type: string
+
+jobs:
+  update-win-version:
+    timeout-minutes: 60
+    runs-on: [self-hosted, windows, x64, vs-2019, vcpkg-2024.10.21, meshinspector]
+    env:
+      vs19_vcpkg_version: ${{ inputs.vs19_vcpkg_version }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v5
+        with:
+          submodules: true
+
+      - name: Checkout Vcpkg ${{env.vs19_vcpkg_version}}
+        working-directory: C:\vcpkg
+        run: |
+          git fetch
+          git checkout ${{env.vs19_vcpkg_version}}
+
+      - name: Restore Vcpkg Cache
+        uses: actions/cache@v4
+        id: vcpkg-cache
+        with:
+          key: vcpkg-cache-${{env.vs19_vcpkg_version}}
+          path: |
+            C:\vcpkg\*
+
+      - name: Update vcpkg packages
+        run: |
+          .\thirdparty\install.bat
```

Comment:
lets add comment that this step is only one keeping us from using github runner for this job